src/v/kafka/server/group.cc

```diff
@@ -2717,6 +2717,11 @@ ss::future<error_code> group::remove() {
               "Replicated group delete record {} at offset {}",
               _id,
               result.value().last_offset);
+        } else if (result.error() == raft::errc::shutting_down) {
+            vlog(
```

Comment:
For shutdown this can probably just be .debug

(in the ticket I hadn't realized that 19 was the shutdown code)